evaluation_framework/README.md

```diff
@@ -0,0 +1,23 @@
+This repository contains modules and driver/test scripts for evaluation of dwi image registration methods
+
+# Requirements
+
+Requirements will increase as more registration methods are added and more evaluation criteria are implemented.
+
+numpy, vtk, nibel, hp5y, ANTsPy, mrtrix, dipy
+
+# Data preparation
+
+While several current driver scripts serve to test the methods starting from dwi, in general the evaluation framework will assume preprocessed data, including:
+
+dti, fa images, brain masks, fod images, and tractography
+
+# Run an example -- ants FA registration and evaluation via fiber tract distance
+
+From inside the scripts directory, run the driver `pairwise_evaluation_ants.py`
+
+```sh
+python pairwise_evaluation_ants.py [-h] [--percent_sample_fibers PERCENT_SAMPLE_FIBERS] [--num_repeats NUM_REPEATS] source source_mask source_fiber_dir target target_mask target_fiber_dir output_dir
+```
+
```

Comment:
Something important is undocumented here: To run the script, you need to input DWI, with a mask, and with tractography results.
- say that the expected file formats for DWI and mask are nifti, with bvals and bvecs provided in FSL format.
- explain how to obtain tractography results. Otherwise starting with just a DWI it is not clear what this wants for tractography results. How do we get the results, and what part of the output exactly are we pointing this script to